thoth/storages/graph/enums.py

```diff
@@ -63,3 +63,10 @@ class MetadataDistutilsTypeEnum(Enum):
     # rarely used fields
     PROVIDED = "PROVIDED"
     OBSOLETE = "OBSOLETE"
+
+
+class QuerySortTypeEnum(Enum):
+    """Class for the requirements format."""
+
+    PACKAGE_NAME = "package_name"
+    PACKAGE_VERSION = "package_version"
```

Comment:
In general, it's better to use int for enum values - comparing ints much less expensive operation than string comparison. Let's address it in a separate PR.